pkgs/tools/security/grype/default.nix

```diff
@@ -59,6 +60,7 @@ buildGoModule rec {
   '';
 
   meta = with lib; {
+    broken = stdenv.isDarwin;
```

Comment:
it's only x86_64 darwin, also please add a comment linking to the issue (see syft)

Reply:
also broken is usually set at the end, after text meta data.

Reply:
> also broken is usually set at the end, after text meta data.

#175472 added it on the top.